src/platform/forms-system/src/js/containers/FormApp.jsx

```diff
@@ -52,7 +53,7 @@ class FormApp extends React.Component {
     // 1. we're not on the intro page *or* one of the additionalRoutes
     //    specified in the form config
     // 2. there is a title specified in the form config
-    if (!isIntroductionPage && !isNonFormPage && title) {
+    if (!isIntroductionPage && !isNonFormPage && !hiddenFormTitle && title) {
```

Comment:
Suuuuper small nitpick but maybe it would be more consistent to name this variable `hasHiddenFormTitle`? For booleans I tend to like naming variables starting with `is...` or `has...`, for legibility.

Reply:
Good catch, fixed! 👍